test/test_gpu/skip_tests_h100_pytorch.yaml

```diff
@@ -7,6 +7,8 @@
 flash_attention:
   # thunderkittens cannot handle the default input shapes
   - tk
+  # tma API changed in upstream
+  - triton_tutorial_flash_v2_tma
```

Comment:
fp8_attention/triton_flash_v2_tma also fails, can you please also add it to the skip list?